src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -106,3 +106,80 @@ function *(c::Number, A::DerivativeOperator{T,N,Wind}) where {T,N,Wind}
         A.low_boundary_coefs,
         A.high_boundary_coefs,coefficients,A.coeff_func)
 end
+
+# Inplace left division
+function LinearAlgebra.ldiv!(M_temp::AbstractArray{T,MT}, A::DerivativeOperator{T,N}, M::AbstractArray{T,MT}) where {T<:Real, N, MT}
```

Comment:
Shouldn't this just be a concretization function, nd ldiv should just call the concretization function? I don't think we need `\` overloads, just good concretizers on the composed methods?

Reply:
@ChrisRackauckas So just scrap `ldiv!` and `\` and instead implement things like `Array` and `SparseMatrixCSC` for higher dimensional operators?

Reply:
Yeah that would work.